### ISSUE TITLE
Would a F# version of the example be desired?

### ISSUE THREAD

**Issue by jeremysmith1** [elastic/elasticsearch-net-example#26]
I would like to make the examples in a F# version. Does this exist already? Would this be desired? I could see it adding confusion as to what language to use.

**Comment by russcam** [comment]
An F# branch would be very cool! You'd likely need to build an F# API layer over some of NEST's constructs to make it more idiomatically F# to work with

**Comment by jeremysmith1** [comment]
Agreed. I am working on bulkall requests. Adding an api layer is a good idea.

Maybe writing that layer would be a good first step.

**Comment by haf-afa** [comment]
Yes please. Also, v7? Seems like `CreateIndex` is gone from the client.

**Comment by Mpdreamz** [comment]
We've namespaced methods since Elasticsearch nowadays has `270+` API's.

`client.Indices.Create()` is the new `client.CreateIndex()`

https://www.nuget.org/packages/NEST.7xUpgradeAssistant/

Can be use to temporarily reintroduce the old methods with obsolete warnings indicating how to rewrite them.

**Comment by russcam** [comment]
There are now 7.x and 7.x-codecomplete branches that have updated the example to 7.x, using ASP.NET Core 3.0.
